packages/cf-component-dropdown/src/Dropdown.js

```diff
@@ -81,4 +128,4 @@ Dropdown.childContextTypes = {
   dropdownRegistry: PropTypes.instanceOf(DropdownRegistry).isRequired
 };
 
-export default Dropdown;
+export default createComponent(styles, Dropdown, ['align']);
```

Comment:
Yea, createComponent checks if you're wrapping a React component and [tries to add its propTypes](https://github.com/cloudflare/cf-ui/blob/master/packages/cf-style-container/src/index.js#L14) into the third parameter.